### DISCUSSION TITLE
NVDA support for GoToMeeting & WebEx

### DISCUSSION THREAD

**Discussion by peterpacult** [nvaccess/nvda#14762] [Ideas]
### Is your feature request related to a problem? Please describe.
The web conferencing systems are not really (or only very limitedly) accessible.

### Describe the solution you'd like
NVDA is to support the GoToMeeting and WebEx desktop apps. Navigation through the app must be ensured; one must be able to access settings.

### Describe alternatives you've considered
There are keyboard shortcuts, but you don't get to see what's happening when you use them. See also: https://nvda.groups.io/g/nvda/topic/using_gotomeeting/73306031?p=,,,20,0,0,0::recentpostdate%2Fsticky,,,20,2,460,73306031

### Additional context
It is important that NVDA also supports these video conferencing programmes because they are often used in administrations and official institutions (for data protection reasons).

**Comment by josephsl** [comment]
Hi, have you tried contacting the app vendor(s) about accessibility and usability with screen readers in addition to NVDA? Thanks.

**Comment by Adriani90** [comment]
cc: @mohammad-suliman you developed the zoom addon for NVDA, is there any cooperation with other app vendors as well?

**Comment by peterpacult** [comment]
Hi Joseph,

I have tried to contact GoToMeeting. Unfortunately, I did not find a proper
support address where I could have described my accessibility problems in
detail.
I have only found the following contact form:
https://www.gotomeeting.com/de-de/webkonferenz-testen
Unfortunately, it seems to be for corporate customers only.

The situation is similar at Webex. I have no way to get in touch with the
developers. The only form I could find does not even allow me to describe a
concern, but only allows me to "order" a product for a company:
https://www.webex.com/de/dg/all-new-webex.html?DTID=pseggI000015&CCID=cc001728&utm_source=GOOG&utm_channel=SEM&utm_campaign=MMEBrand&utm_medium=GER&utm_content=MMENTEM&utm_term=Webex&gclsrc=aw.ds&&gclid=Cj0KCQjwi7yCBhDJARIsAMWFScPd15I7JI6XfuZZPonlyQCApjpYwoTUwNl9JeJflB0NW8VmAON2ED0aAtaREALw_wcB#formAnchor

For example, Webex/GoToMeeting is often used in German schools or in
administration. This has to do with data protection aspects in Europe.

With kind regards

Niels

Am Mo., 8. März 2021 um 10:50 Uhr schrieb Joseph Lee <
***@***.***>:

**Comment by peterpacult** [comment]
There are still problems with the operation of GoToMeeting. It would be nice if an NVDA developer could take another look at it!

**Comment by mohammad-suliman** [comment]
Thanks @Adriani90 ! In fact, we had only considered Zoom for our add-on development efforts, and we hadn't any cooperation with other communication software venders. We of course contacted Zoom for the accessibility issues we tried to tackle, but only them as I have said before. I don't know if we will have the chance to tackle another video communication programs in the future.

**Comment by Adriani90** [comment]
Î will convert this to a discussion in order to exhange global argument on video conferencing software. If you have very specific issues, then create new single issues for your problems so that they can be solved by pull requests.
